woocommerce/woocommerce
# Add NX commands for Jetpack Changelogger

### All Submissions:






### Changes proposed in this Pull Request:

Add NX commands for [Jetpack Changelogger](https://github.com/Automattic/jetpack/tree/master/projects/packages/changelogger).

Part of https://github.com/woocommerce/woocommerce/issues/31135

### How to test the changes in this Pull Request:

1. Add a changelog entry
```
nx run woocommerce:changelog --action add
```
2. Validate your entry

```
nx run woocommerce:changelog --action validate
```
3. Write your entry to the changelog. Note the use of `--use-version` is because the changelog has no previous entry to bump from.

```
nx run woocommerce:changelog --action write --use-version=99.0
```
4. Try a nonsensical command to see the error appropriately displayed

```
nx run woocommerce:changelog --action yadadadada
```

### Other information:




### FOR PR REVIEWER ONLY:
